src/node/entry.ts

```diff
@@ -126,19 +122,40 @@ const main = async (cliArgs: Args): Promise<void> => {
   }
 }
 
+function trimLDLibraryPath(): void {
+  let ldVar: string
+  if (process.platform === "linux") {
```

Comment:
I think we might want to check if the variable exists or something rather than use the platform because `process.platform` returns non-`linux` values even if `uname` normally returns `Linux`. https://nodejs.org/api/process.html#process_process_platform

Reply:
Actually that might only be the case for Android. :thinking: The rest seem to match up in terms of `uname` and `process.platform`.